### ISSUE TITLE
invert_axes, invert_xaxis, and invert_yaxis are named and documented in a misleading way

### ISSUE THREAD

**Issue by foobarbecue** [ioam/holoviews#2743]
The invert_xaxis and invert_yaxes options, as expected, flip the axes when set to true, so that they go from large to small left to right and down to up, opposite of the conventional bottom left small-number origin.

invert_axes, however, does something totally different and unexpected -- it swaps the x and y axes. The docstrings make it sound like setting invert_axes=True is the same as setting invert_xaxis=True and invert_yaxis=True at the same time:

    invert_axes = param.Boolean(default=False, doc="""
        Whether to invert the x- and y-axis""")

    invert_xaxis = param.Boolean(default=False, doc="""
        Whether to invert the plot x-axis.""")

    invert_yaxis = param.Boolean(default=False, doc="""
        Whether to invert the plot y-axis.""")

(holoviews/plotting/plot.py line 585 to 592)

At a minimum, the docstring for invert_xaxes should be changed to "Whether to swap the x- and y- axes." Even clearer would be to rename invert_axes to swap_axes, but I realize this could be a hassle given backwards compatibility requirements.

**Comment by jlstevens** [comment]
I think I am in favor of this suggestion: add ``swap_axes`` and support ``invert_axes`` for backwards compatibility with a plan to deprecate it. So one release encouraging ``swap_axes`` and using it in the docs, a subsequent release where ``invert_axes`` issues deprecation warnings (that can be disabled via ``hv.config`` if requested) before finally deprecating ``invert_axes``. I do agree it is confusing.

**Comment by philippjfr** [comment]
I agree, and would also consider renaming ``invert_x/yaxis`` to ``flip_x/yaxis``.

**Comment by jlstevens** [comment]
Other than 'flip' being shorter than 'invert', what is the benefit of ``flip``? To me, the main issue here is how ``invert_axes`` looks very similar to the other two options but does something different.

**Comment by philippjfr** [comment]
It's much clearer imo.

**Comment by jlstevens** [comment]
Personally, I do prefer ``flip`` but it is such a mild preference I'm not sure it is worth the confusion that would be caused by changing it. For holoviews 2.0, I would be very happy for such an API change though.

**Comment by jbednar** [comment]
I vote for changing `invert_axes` to `swap_axes` to avoid confusion, and for *not* changing `invert_xaxis` and `invert_yaxis`, again to avoid confusion (as once we have `swap_axes` there's less chance of confusion.
